R/bcdc_search.R

```diff
@@ -80,9 +80,9 @@ bcdc_list_groups <- function() bcdc_search_facets("groups")
 bcdc_list_group_records <- function(group) {
   if(!has_internet()) stop("No access to internet", call. = FALSE) # nocov
 
-  cli <- bcdc_catalogue_client("action/group_show")
+  cli <- bcdc_catalogue_client("action/group_package_show")
 
-  r <- cli$get(query = list(id = group, include_datasets = 'true'))
+  r <- cli$get(query = list(id = group, limit = 1000))
```

Comment:
Similar to other ones, do you think we should expose this as an argument in `bcdc_list_group_records()`?